### PR TITLE
Renovation: Implement default button behavior

### PR DESCRIPTION
**Pull request by LazyLahtak**
Implemen base behavior of button:
1) hoverStateEnabled = true
2) activeStateEnabled = true
3) possible to click on button without `accessKey` property

Will be compatible with focus (#11711) after generator fix.

### CODE REVIEW

**Comment by MaximKudriavtsev** [comment]
Should we add some tests?

File: js/renovation/widget.tsx [hunk 1]
```diff
@@ -269,6 +268,18 @@ export default class Widget {
...
+    clickEffect() {
+        const namespace = 'UIFeedback';
+
+        dxClick.on(this.widgetRef,
+            () => this.onClick!(this.clickArgs),
+            { namespace }
+        );
+
+        return () => dxClick.off(this.widgetRef, { namespace });
+    }
```
Comment (MaximKudriavtsev) [inline js/renovation/widget.tsx]: - should we make the subscription without onClick property?
- should we combine `namespace` with `thus.name`?

Comment (LazyLahtak) [reply js/renovation/widget.tsx]: 1) Widget (and Button) still work even if we won't pass `onClick`. Do you mean that?
2) You mean that we should make  'namespace = `UIFeedback${this.name}`'?
Also, the 'this.name' at now is always `Widget` (even if it is `Button`). I think we should fix it.

Comment (MaximKudriavtsev) [reply js/renovation/widget.tsx]: About 2
image
Original button code. namespace = this.name

**Review by MaximKudriavtsev — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Pull request merged**